src/include/function/string_functions.h

```diff
@@ -74,6 +74,12 @@ class StringFunctions {
   // Length will return the number of characters in the given string
   static uint32_t Length(executor::ExecutorContext &ctx, const char *str,
                          uint32_t length);
+
+  // Nextval will return the next value of the given sequence
+  static uint32_t Nextval(executor::ExecutorContext &ctx, const char *sequence_name);
```

Comment:
These two functions, Nextval() and Currval() should be put into a new sequence file and update binder to bind the file.